sphinx_gallery/gen_gallery.py

```diff
@@ -191,32 +184,33 @@ def sumarize_failing_examples(app, exception):
         return
 
     gallery_conf = app.config.sphinx_gallery_conf
-    failing_examples = set([os.path.normpath(path) for path in
-                            gallery_conf['failing_examples']])
-    expected_failing_examples = set([os.path.normpath(path) for path in
+    failing_examples = gallery_conf['failing_examples']
```

Comment:
I don't think you reverted all the unnecessary changes. Keep failing_examples as it was previously.

Keep the `\n\nExamples` if you wish.